operating_versioned_docs/version-v0.5/manuals/collaset.md

```diff
@@ -0,0 +1,968 @@
+collaset-sample   3         3         3         3               3                   collaset-sample-6d7b7c58f   collaset-sample-6d7b7c58f   64s
+```
+
+By default, CollaSet always creates new Pods using the latest template specified in `spec.template`. CollaSet establishes ownership over a set of Pods through the label selector defined in `spec.selector`. Thus, it's important to ensure that the labels provided in `spec.selector` match those in `spec.template.metadata.labels`.
+
+CollaSet status provides general information about this CollaSet and all Pods under it.
+
+``` shell
+$ kubectl -n default get cls collaset-sample -o yaml
+......
+status:
+  availableReplicas: 3
+  collisionCount: 0
+  conditions:
+  - lastTransitionTime: "2023-09-01T03:56:09Z"
+    reason: Updated
+    status: "True"
+    type: Update
+  currentRevision: collaset-sample-6d7b7c58f
+  observedGeneration: 1
+  operatingReplicas: 0
+  readyReplicas: 3
+  replicas: 3
+  scheduledReplicas: 3
+  updatedAvailableReplicas: 3
+  updatedReadyReplicas: 3
+  updatedReplicas: 3
+  updatedRevision: collaset-sample-6d7b7c58f
+```
+
+Some fields in CollaSet status are explained here:
+
+`updatedRevision` indicates the latest revision that CollaSet uses to create or update Pods.
+
+`currentRevision` indicates the last updated revision. It will be set to updatedRevision after all Pods are updated, and their PodReady conditions become True.
+
+`replicas` indicates the count of Pods under this CollaSet.
+
+`scheduledReplicas` indicates the count of Pods under this CollaSet that successfully got scheduled.
+
+`availableReplicas` indicates the count of Pods under this CollaSet that have all expected finalizers attached.
+
+`updatedReplicas` indicates the count of Pods under this CollaSet that have the updated revision.
+
+`updatedReadyReplicas` indicates the count of Pods under this CollaSet that are counted in `updatedReplicas` and have their PodReady conditions set to True.
+
+`updatedAvailableReplicas` indicates the count of Pods under this CollaSet that is counted in `updatedReadyReplicas` and have all expected finalizers attached.
+
+### Updating Pods
+CollaSet generates Pods according to the pod template described in `spec.template`. This template can be updated to signal CollaSet to update each owned Pod:
+
+``` shell
+$ kubectl -n default edit cls collaset-sample
+apiVersion: apps.kusionstack.io/v1alpha1
+kind: CollaSet
+metadata:
+......
+spec:
+......
+  template:
+    ......
+    spec:
+      containers:
+      - image: nginx:1.24.0  # changed from nginx:1.25.2 
+......
+```
+
+CollaSet immediately updates all Pods it owns with the new Pod template by default.
+
+``` shell
+$ kubectl -n default get pod -o yaml | grep "image: nginx"
+    - image: nginx:1.24.0
+    - image: nginx:1.24.0
+    - image: nginx:1.24.0
+```
+
+The update progress can be controlled using partition.
+
+#### Partition
+Similar to StatefulSet, `partition` is used to control the  progress.
```

Comment:
multi space between "the process".
ps, in v0.4 “the upgrade process" also works

Reply:
fixed: Similar to StatefulSet, `partition` is used to control the upgrade progress.